nzedb/Binaries.php

```diff
@@ -1308,6 +1398,22 @@ private function addMissingParts($numbers, $tableName, $groupID)
 	}
 
 	/**
+	 * Add article numbers from missing headers to DB.
+	 *
+	 * @param array  $numbers   The article numbers of the missing headers.
+	 *
+	 * @return bool
+	 */
+	private function addMissingMGRParts($numbers)
```

Comment:
add 'array' to the definition if it MUST be an array. This causes an error if non-arrays are passed making debugging easier